conda_lock/src_parser/lockfile.py

```diff
@@ -0,0 +1,30 @@
+import pathlib
+
+import toml
```

Comment:
I wanted to say something similar. It would be great to keep this YAML because we could implement a parser in C++ as well pretty easily for `micromamba` (we are already using yaml-cpp).

Reply:
Okay, I'm sold; YAML it is.